cmd/completion.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright © 2020 The PES Open Source Team pesos@pes.edu
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This is where you've got extra white space.

Reply:
Finally caught it. Tabs vs whitespace